### PR TITLE
murano: add murano proposal

### PR DESCRIPTION
**Pull request by Itxaka**

### CODE REVIEW

**Comment by dirkmueller** [comment]
+1

File: scripts/qa_crowbarsetup.sh [hunk 1]
```diff
@@ -3296,6 +3297,13 @@ function deploy_single_proposal()
                 return
             fi
             ;;
+        murano)
+            [[ -n "$want_murano" ]] || return
```
Comment (jdsn) [inline scripts/qa_crowbarsetup.sh]: if you use `[[ ]]` then you don't need `-n` and quotes
So easier and shorter:
`[[ $want_murano ]] || return`

**Comment by toabctl** [comment]
+1

File: scripts/qa_crowbarsetup.sh (hunk 1 above)
Comment (bmwiedemann) [inline scripts/qa_crowbarsetup.sh]: made related PR #1210

**Pull request merged**